css/properties/appearance.json

```diff
@@ -250,34 +226,50 @@
                   "partial_implementation": true
                 }
               ],
-              "firefox_android": {
-                "version_added": "4",
-                "partial_implementation": true
-              },
+              "firefox_android": [
+                {
+                  "version_added": "80"
+                },
+                {
+                  "version_added": "4",
+                  "partial_implementation": true
+                }
+              ],
               "ie": {
                 "version_added": false
               },
-              "opera": {
-                "version_added": "15"
-              },
+              "opera": [
+                {
+                  "version_added": "69"
+                },
+                {
+                  "version_added": "15",
+                  "partial_implementation": true
+                }
+              ],
               "opera_android": {
-                "version_added": "14"
+                "version_added": "14",
```

Comment:
Can you add the non-`partial_implementation` entry here as well? If you run `npm run mirror -- --modify=always opera_android css` it will be among the resulting changes, but you may have to sift through some other changes. If they seem correct, feel free to throw them into this PR as well though.